docs/source/workbench/superplot.rst

```diff
@@ -0,0 +1,41 @@
+.. _WorkbenchSuperplot:
+
+=========
+Superplot
+=========
+
+Superplot is a decorator widget of the plot window. It facilitates over-plotting
+and manipulation of overplotted data. The superplot is accessible:
+
+* Directly in the plot window through the menu bar
+* In the context menu of supported workspaces
+
+
+**Widget description**
+
+The superplot widget is composed of two distinct parts:
+
+* Left side of the plot window: a list of selected workspaces and spectra. This
+  list contains workspaces names and under them the plotted data if any.
+* Bottom of the plot: a navigation bar that contains a slider and different
+  buttons to navigate through the bins/spectra and control the plotted data.
+
+.. image:: ../images/superplot_1.png
+   :align: center
+
+
+**Usage**
+
+When the superplot is activated, its state is updated based on the current plot.
+The list will contain the names of the selected workspaces and below each of
+them, the plotted bins/spectra. One can then add a workspace in the list by
+dragging and dropping it in the list or by using the combobox.
+
+When one or many workspace(s) is(are) clicked in the list, the navigation bar
+can be used to slide over the bins/spectra and update the plot accordingly. One
+can then add the bin/spectrum currently pointed by the slider to the list by
+pressing the hold button.
+
+Each workspace and bin/spectrum is associated, in the list, with a delete
+button. To remove a curve, one can also slide to the specific position and
+release the hold button.
```

Comment:
This text misses the main point, which is that the current curve indicated by the slider for the workspaces that are selected from the list is plotted, but not persisted. One can move the slider to iterate through the curves one by one, and, if needed, click Hold to persist a curve on the plot. 
Also, for the raw data, the legend shows the spectrum number, whereas the spinbox in superplot refers to the workspace index, as it should be. The index always starts from 0 and is contiguous, whereas the spectrum number starts from 1 and might not be contiguous if there are grouped detectors. So for raw data the index indicated in the spinbox will often be 1 less than the spectrum number in the legend/workspace list. This has to be highlighted in the docs.

Reply:
done